oximeter/db/src/client/mod.rs

```diff
@@ -960,6 +984,125 @@ impl Client {
         }
         Ok(())
     }
+
+    /// Given a list of timeseries by name, delete their schema and any
+    /// associated data records from all tables.
+    async fn expunge_timeseries_by_name(
+        &self,
+        replicated: bool,
+        to_delete: &[TimeseriesName],
+    ) -> Result<(), Error> {
+        // The version table should not have any matching data, but let's avoid
+        // it entirely anyway.
+        let tables = self
+            .list_oximeter_database_tables(ListDetails {
+                include_version: false,
+                replicated,
+            })
+            .await?;
+
+        const DELETE_BATCH_SIZE: usize = 1000;
```

Comment:
any particular reasoning for the `1000` batch size for chunking? Maybe a comment at least.

Reply:
Not really, it's a WAG. I wanted to limit request sizes somewhat, and this will keep us below a few 10s of KiB for the most part. I added a comment to that effect in c9bd27c10.